src/clusterfuzz/environment/__init__.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Environment helpers."""
+
+import contextlib
+import os
+
+from clusterfuzz._internal.system import environment
+
+# Inverted SANITIZER_NAME_MAP.
+SANITIZER_MAP = {v: k for k, v in environment.SANITIZER_NAME_MAP.items()}
```

Comment:
Should we assert that each value in SANITIZER_NAME_MAP is unique?

Reply:
Good point, added!